### ISSUE TITLE
[bug] fixture_path returns nil in 3.7.2

### ISSUE THREAD

**Issue by cdchow** [rspec/rspec-rails#1921]
Hi,

I'm currently using `rspec-rails` 3.7.1 and tried bumping up to 3.7.2 only to learn that a small handful of my specs failed. In particular, `fixture_file_upload` throws an error where it could not find the file: 
```
Failure/Error: let(:file) { fixture_file_upload("spec/fixtures/report.txt", "image/png") }     

RuntimeError:
  /spec/fixtures/report.txt file does not exist
```

I'm on `ruby 2.3.3p222` and `rails (= 4.2.8)`. I believe this may be related to #1880 which exists in 3.7.2 but not 3.7.1 (when I searched the tags). Please let me know if any additional information is required.

Thanks.

**Comment by 11factory** [comment]
`/spec/fixtures/report.txt` heading `/` makes me think that `fixture_path` has not been set properly. Can you check that please ? [](https://apidock.com/rails/ActionDispatch/TestProcess/fixture_file_upload)

**Comment by bmulholland** [comment]
@11factory fixture_path is nil with both 3.7.1 and 3.7.2. It looks like the ActionDispatch version of fixture_file_upload does not prefix a "/" when `fixture_path` is nil. However, in that case the code from #1880 effectively runs `fixture_path = Rails.join("", "")` which resolves to "/".

**Comment by 11factory** [comment]
a fix is on PR https://github.com/rspec/rspec-rails/pull/1943

**Comment by mikeycgto** [comment]
Is this still an issue? I see to be experiencing this problem in a Rails 5.0 application with rspec-rails 3.7.2. I'm able to work around it with the following monkey patch:

```ruby
class RSpec::Core::ExampleGroup
  def self.fixture_path
     Rails.root.join('spec', 'fixtures', 'files').to_s
  end
end
```

Without this patch, I get the following error:

```
RuntimeError:
  image.png file does not exist
```

When doing:

```ruby
let(:content_screen_shot) { fixture_file_upload('image.png', 'image/png') }
```

Seems like `fixture_file_upload` fails to find a `fixture_path` class method. Any advice?

**Comment by bmulholland** [comment]
@mikeycgto looks like it's fixed on master but not yet released: https://github.com/rspec/rspec-rails/blob/master/Changelog.md "Use relative path for resolving fixtures when fixture_path is not set. (Laurent Cobos, #1943)"

**Comment by samphippen** [comment]
This is fixed by #1943 and will be released in the next RSpec Rails version.
